docs/develop/unified-manifest-overview.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Office Add-ins with the unified app manifest for Microsoft 365
-description: Get an overview of the unified app manifest for Microsoft 365 for Office Add-ins and its uses.
+description: Gets an overview of the unified app manifest for Microsoft 365 for Office Add-ins and its uses.
```

Comment:
Hmm. I thought our "descriptions" are supposed to be imperative mood sentences. For example, think of all the descriptions that begin "Learn ... "

Reply:
Sorry about that. I was in Script Lab mode. Will undo.